plugins/mcreator-localization/help/fr_FR/structure/ignore_blocks.md

```diff
@@ -1,5 +1,3 @@
-This option will specify which blocks not to place (ignore) when generating the structure.
+Cette option spécifie quels blocs ne pas poser (ignorer) lors de la génération de la structure.
 
-Select air here in order to not place air blocks of your structure. This will make the structure
-integrate better with the environment, but in case of cave based structures, you will likely want to
-place the air too.
\ No newline at end of file
+Sélectionnez Air pour ne pas poser les blocs d'air de la structure. Cela intègrera mieux la structure dans son environnement, mais dans le cas où la structure est une cave, placer l'air est requis.
```

Comment:
Cave = grotte ou caverne

Reply:
Indeed. French "Cave" is a part of the house.

Reply:
Fixed too!